dynamite_flask_app/resources/product.py

```diff
@@ -0,0 +1,61 @@
+"""Product resource REST API module."""
+
+from flask import request, jsonify
+from flask_restful import Resource
+from flask_jwt_extended import jwt_required
+
+from models import Product, Size
+from schemas import product_schema, products_schema
+
+
+class Products(Resource):
+    """Creates Product resource that uses the Porduct model for required results."""
+
+    def get(self):
+        """May take multiple arguments provided in the API. Also has pagination functionality.
+        Will also work without any arguments.
+
+        Args:
+            size(str, optional): The porduct size that needs to be searched.
```

Comment:
These arguments are for the api request not for the function. To document API use a different method than the one used for doc strings.